rhombus/scribblings/ref-pair.scrbl

```diff
@@ -103,3 +127,399 @@ its ``rest'' is a list.
 )
 
 }
+
+
+@doc(
+  annot.macro 'PairList'
+  annot.macro 'PairList.of($annot)'
+){
+
+ Matches any @tech{pair list} in the form without @rhombus(of). The @rhombus(of)
+ variant matches a pair list whose elements satisfy @rhombus(annot).
+
+ Static information associated by @rhombus(PairList, ~annot) or
+ @rhombus(PairList.of, ~annot) makes an expression acceptable as a sequence
+ to @rhombus(for) in static mode.
+
+}
+
+@doc(
+  ~nonterminal:
+    listable_expr: block expr
+  fun PairList(v :: Any, ...) :: List
+  expr.macro 'PairList [$expr_or_splice, ...]'
```

Comment:
This should exclude the space to be consistent with `List`.

Reply:
HF